### ISSUE TITLE
olm-openshift: operands end up in crashloop

### ISSUE THREAD

**Issue by frzifus** [grafana/tempo-operator#363]
```bash
NAME                                             READY   STATUS             RESTARTS      AGE
minio-5794f6799d-pl6hn                           1/1     Running            0             25h
tempo-simplest-compactor-549ffd7b5c-z2db6        0/1     CrashLoopBackOff   3 (22s ago)   66s
tempo-simplest-distributor-5654468b76-llvcv      0/1     CrashLoopBackOff   3 (9s ago)    66s
tempo-simplest-gateway-67fd7f85bd-h7jhj          2/2     Running            0             65s
tempo-simplest-ingester-0                        0/1     CrashLoopBackOff   2 (32s ago)   66s
tempo-simplest-querier-fcbfbffd7-75pkl           0/1     CrashLoopBackOff   3 (9s ago)    66s
tempo-simplest-query-frontend-85bbb96575-wz7fj   0/2     CrashLoopBackOff   3 (21s ago)   66s
```

Used the following to create the tempo instance:
```yaml
apiVersion: tempo.grafana.com/v1alpha1
kind:  TempoStack
metadata:
  name: simplest
spec:
  storage:
    secret:
      name: minio-test
      type: s3
  storageSize: 1Gi
  resources:
    total:
      limits:
        memory: 2Gi
        cpu: 2000m
  tenants:
    mode: openshift
    authentication:
      - tenantName: dev
        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfa"
      - tenantName: prod
        tenantId: "1610b0c3-c509-4592-a256-a1871353dbfb"
  template:
    gateway:
      enabled: true
    queryFrontend:
      jaegerQuery:
        enabled: true
```

All tempo logs show:
```bash
$ k logs -n summitdemo tempo-simplest-ingester-0
failed parsing config: failed to parse configFile /conf/tempo.yaml: yaml: unmarshal errors:
  line 52: field internal_server not found in type app.Config
```

Bundle was build using:
```bash
BUNDLE_VARIANT=openshift OPERATOR_VERSION=0.0.9 USER=frzifus make bundle docker-build docker-push docker-push bundle-build bundle-push catalog-build catalog-push
```

Version: https://github.com/os-observability/tempo-operator/commit/5ec7feaa09e5f4ce224732a148b1ca334c841d0c

---

Update:

**tempo.yaml**:
```bash
    48	      tls_min_version: VersionTLS13
    49	  search:
    50	    external_hedge_requests_at: 8s
    51	    external_hedge_requests_up_to: 2
    52	internal_server:
    53	  enable: true
    54	  http_listen_address: ""
    55	  tls_cipher_suites: 
    56	  tls_min_version: VersionTLS13
    57	  http_tls_config:
    58	    cert_file: /var/run/tls/http/server/tls.crt
    59	    key_file: /var/run/tls/http/server/tls.key
    60	server:
    61	  grpc_server_max_recv_msg_size: 4194304
    62	  grpc_server_max_send_msg_size: 4194304
```

**Comment by rubenvp8510** [comment]
This is due a feature added to tempo that is not released yet. and is required to make the readiness probe works with mTLS.

**Comment by frzifus** [comment]
Can be fixed using:
```yaml
spec:
  images:
    tempo: docker.io/grafana/tempo:main-e6394c3
    tempoQuery: docker.io/grafana/tempo-query:main-e6394c3
    tempoGateway: quay.io/observatorium/api:main-2023-02-09-v0.1.2-329-g1ff4f11
```

**Comment by andreasgerstmayr** [comment]
Should be fixed with #408
